src/avfoundation.cs

```diff
@@ -796,31 +797,41 @@ interface AVAudioFormat : NSSecureCoding {
 
 		[Export ("isEqual:"), Internal]
 		bool IsEqual (NSObject obj);
+		
+		[iOS (10,0), TV (10,0), Watch (3,0), Mac (10,12)]
+		[NullAllowed, Export ("magicCookie", ArgumentSemantic.Retain)]
+		NSData MagicCookie { get; set; }
 	}
 
 	[Protocol, Model]
 	[BaseType (typeof (NSObject))]
 	interface AVAudio3DMixing {
+		[NoWatch]
```

Comment:
Sadly it's a common Apple pattern (to mark the selectors and not the type itself), even more for existing types (I wonder if it's a tooling issue on their side).

In such cases don't worry about disabling the type as it won't be a breaking change to introduce it later (but we won't be able to remove it without causing a breaking change).

As for the API just disabling the type will tell you (if it compiles or not) if it's needed (or not). If that happens then file a rdar because it does not mean the type is allowed - see https://trello.com/c/4soJeYEr